### PR TITLE
Fix accumulo-cluster remote start error

### PR DESCRIPTION
**Pull request by dlmarion**

### CODE REVIEW

File: assemble/bin/accumulo-cluster [hunk 1]
```diff
@@ -129,9 +129,21 @@ function control_service() {
...
+      if [[ $# -gt 3 ]]; then
+        ACCUMULO_SERVICE_INSTANCE="${ACCUMULO_SERVICE_INSTANCE}" "${bin}/accumulo-service" "$service" "$control_cmd" "${@:4}"
+      else
+        ACCUMULO_SERVICE_INSTANCE="${ACCUMULO_SERVICE_INSTANCE}" "${bin}/accumulo-service" "$service" "$control_cmd"
+      fi
     else
-      $SSH "$host" "bash -c 'ACCUMULO_SERVICE_INSTANCE=${ACCUMULO_SERVICE_INSTANCE} ${bin}/accumulo-service \"$service\" \"$control_cmd\" \"${@:4}\" '"
+      if [[ $# -gt 3 ]]; then
+        EXTRA_ARGS="${@:4}"
+        $SSH "$host" "bash -c 'ACCUMULO_SERVICE_INSTANCE=${ACCUMULO_SERVICE_INSTANCE} ${bin}/accumulo-service \"$service\" \"$control_cmd\" $EXTRA_ARGS '"
```
Comment (ctubbsii) [inline assemble/bin/accumulo-cluster:143]: This is flattening the array into a single string, which can mess with our ability to reason about the quoting. It's probably better to be consistent, and use the array form (or leave the original, which was already an array):
```suggestion
        EXTRA_ARGS=("${@:4}")
        $SSH "$host" "bash -c 'ACCUMULO_SERVICE_INSTANCE=${ACCUMULO_SERVICE_INSTANCE} ${bin}/accumulo-service \"$service\" \"$control_cmd\" \"${EXTRA_ARGS[@]}\" '"
```

**Review by ctubbsii — CHANGES_REQUESTED** [review]

File: assemble/bin/accumulo-cluster (hunk 1 above)
Comment (dlmarion) [reply assemble/bin/accumulo-cluster:143]: I can test this out and see if it works.

Comment (dlmarion) [reply assemble/bin/accumulo-cluster:143]: So, the args come over as one string in the way that you suggested and does not work. Error in the compactor.err file:

```
Was passed main parameter '-q q2' but no main parameter was defined in your arg class
```

Comment (ctubbsii) [reply assemble/bin/accumulo-cluster:143]: Okay, that's fine then. It looks like it's flattening them already on the local end. So, I guess we need to pass them unquoted so they get treated as separate args... or we could do something crazy like build the command using an array, but that seems like overkill. I think your solution that worked without the quoting is fine.

**Review by ctubbsii — APPROVED** [review]

**Pull request merged**